source/shared/net/guides/deciding-wallet.rst

```diff
@@ -1,79 +1,100 @@
+- **Desktop Wallet**: Choose this wallet if you want extra security in the form of a :ref:`Ledger<install-Ledger-app>` hardware device, want to create multi-signature transactions, and want to be able to create a baker account.
+
+- **Mobile Wallet**: Choose this wallet if you want to be able to access your accounts on the go, don’t want to run a node, don’t want to depend on a Ledger hardware device, and only need to create standard (single signature) transactions.
+
+The following provides more details on the differences between the wallets.
+
+How the wallets store your private keys
+=======================================
+
+One of the main differences between the two wallets lies in how they store your private keys.
+
+-  The Desktop Wallet stores your private keys on a Ledger that is secured by a PIN code and backed up by a recovery phrase. The private keys never leave the Ledger hardware device, and the Ledger doesn't require internet access, which means you can keep your private keys offline. However, this also means that you need the Ledger to sign the transactions you create in the Desktop Wallet, which makes it less convenient than the Mobile Wallet.
+
+- The Mobile Wallet creates and stores your private keys on the phone. This means it's easier to use because you don't need a Ledger device to make transactions. However, because your phone is connected to the internet, the wallet is more vulnerable to security breaches than the Desktop Wallet. Also, anyone with access to the mobile wallet can make transfers from the accounts in the mobile wallet.
 
-**Mobile Wallet**: you’ll need an iPhone running iOS 13 or later or an Android phone running Android 8 or later.
+How to backup a wallet
+======================
 
-**Desktop Wallet**: you’ll need a computer running Windows, macOS or Linux, and you’ll need a Ledger Nano S. You’ll also have to run a node, and your computer must meet the hardware requirements for this. Go to the :ref:`downloads <downloads>` page for more information.
+We strongly recommend that you make a backup of your wallet regardless of which one you use. Backups are created in different ways in the two wallets.
 
-Why do I need a Ledger hardware device to use the Desktop Wallet?
-==================================================================
+- **Backup of the Desktop Wallet**: You create a backup of your accounts, identities, and addresses by exporting them to a file from the Desktop Wallet. This doesn't mean that you create a backup of your private keys. The backup of your private keys is essentially the 24-recovery phrase for the Ledger. If you lose the PIN code to the Ledger, you can restore the Ledger device from your recovery phrase. You can also set up a new Ledger device with the recovery phrase. It's vital that you keep the recovery phrase safe. For more information, see :ref:`Make a backup of identities, accounts, and addresses<export-import-desktop>` and :ref:`Account recovery<account-recovery-desktop>`.
```

Comment:
Something missing here: "24-recovery"
Maybe 24-word recovery phrase or just seed phrase as Ledger calls it.

Reply:
Ledger uses both terms, but I think they use recovery phrase more than seed phrase.